appscale/tools/appscale_tools.py

```diff
@@ -818,6 +818,15 @@ def terminate_instances(cls, options):
     except (IOError, AppScaleException):
       # Don't fail if we cannot find the configuration.
       pass
+    except AppControllerException as e:
+      # If we are terminating a cloud infrastructure, we should log and
+      # continue.
+      if (infrastructure in InfrastructureAgentFactory.VALID_AGENTS and
+            options.terminate):
+        AppScaleLogger.warn(e)
```

Comment:
No.. It isn't really sensible to not perform terminate of AppScale services for cloud, because at least `RemoteHelper.terminate_virtualized_cluster` also can do clean, if clean flag is specified.